tourney/achievements/achievements.py

```diff
@@ -14,6 +14,11 @@
 from .flawless_defeat_achievement import FlawlessDefeatAchievement
 from .reporter_achievement import ReporterAchievement
 from .leave_channel_achievement import LeaveChannelAchievement
+from .firstJoinerAchievement import FirstJoinerAchievement
```

Comment:
I'd like the file name to use the same pythonic way as the others: `first_joiner_achievement`

Reply:
Yeah, good point.